### PR TITLE
Removed the field end_date from the event table.

### PR DESCRIPTION
**Pull request by marioabreu**
Added a grid to center the events on the page.
Added a Form Component file to add layout to forms.

### CODE REVIEW

File: app/assets/stylesheets/components/_form.sass [hunk 1]
```diff
@@ -0,0 +1,13 @@
+$Form-spacing-small: $Theme-spacing-small
+$Form-spacing-base: $Theme-spacing-base
+
+.Form
+
+.Form-item
+  padding: $Form-spacing-small
+
+.Form-field
+  padding: 10px
```
Comment (marioabreu) [inline app/assets/stylesheets/components/_form.sass:10]: This value is not supposed to be here. I'm going to change it to Form-spacing-small

File: app/assets/stylesheets/components/_grid.sass [hunk 2]
```diff
@@ -0,0 +1,9 @@
+$Grid-margin-base: $Theme-spacing-base
+
+.Grid
+  margin-top: $Grid-margin-base
```
Comment (azevedo) [inline app/assets/stylesheets/components/_grid.sass:4]: Maybe you meant to have these 2 margins as negative instead?

File: spec/models/event_spec.rb [hunk 3]
```diff
@@ -9,9 +9,10 @@
         Availability.create(week_day: 5, start_time: DateTime.new(2015, 03, 06, 20, 00), end_time: DateTime.new(2015, 03, 06, 23, 30), user_id: user.id)
         event = Event.create(name: 'Random Event', location: 'Braga', start_date: DateTime.new(2015, 03, 06, 20, 30), end_date: DateTime.new(2015, 03, 06, 21, 00), owner: user.id, interest_list: 'Ruby')
 
-        result = event.notify_users
+        allow(event).to reveice(:notify_users)
```
Comment (azevedo) [inline spec/models/event_spec.rb]: How come this passes? There's a typo

File: spec/models/event_spec.rb [hunk 4]
```diff
@@ -9,9 +9,10 @@
...
+        event.notify_users
 
-        expect(result).to eq([user])
+        expect(event).to have_received(:notify_users)
```
Comment (azevedo) [inline spec/models/event_spec.rb:15]: This whole test doesn't make sense.
1. You say you're testing the instance method _users_available_ and you're not even calling the method
2. You say you're testing that a list is returned with a user, but you're not testing against any return value
3. There's no point in writing a test that does this: "I'm going to test that if i call X on Y, then X should have been called on Y"

File: app/views/users/_fields.slim [hunk 5]
```diff
@@ -1,5 +1,19 @@
-= f.text_field :name
-= f.text_field :location
-= f.text_field :gender, collection: [:female, :male]
-= f.text_field :interest_list, value: @user.interest_list.join(',')
-= f.submit
+.Form
+  .Form-item
+    = f.label :name
+  .Form-item--input
```
Comment (azevedo) [inline app/views/users/_fields.slim:4]: 1. Let's try to use modifiers only in the main Component
2. This modifier isn't defined in the stylesheet

File: app/views/events/index.slim [hunk 6]
```diff
@@ -2,10 +2,12 @@ h2 Events
 -@events.each do |event|
   = link_to event.name, event_path(event.id)
   ul
-    li 
+    li
       b = "On #{Date::DAYNAMES[event.start_date.wday]}"
     li = event.starting_date
-    li = event.ending_date
       li.fa.fa-user = "#{event.attendees.size}"
+    -unless current_user.nil?
```
Comment (azevedo) [inline app/views/events/index.slim]: Couldn't you pass `current_user` into `event.attending?`?
Or if that's not doable, maybe you could write that predicate in one line `- if current_user.present? && event.attending?(current_user.id)`

File: spec/models/event_spec.rb (hunk 4 above)
Comment (marioabreu) [inline spec/models/event_spec.rb:15]: I have to review this test, this is failing for some time now.
The idea is to test id the method is calling notify users is called.

File: app/views/events/index.slim (hunk 6 above)
Comment (marioabreu) [inline app/views/events/index.slim]: Yes i can.
I'm going with your first recommendation.
Basically this check is useless.

File: app/views/users/_fields.slim (hunk 5 above)
Comment (marioabreu) [inline app/views/users/_fields.slim:4]: I removed it.
@fcbaila is already on the design, so i will do the forms part later.

**Comment by marioabreu** [comment]
I think i will remove this branch, because i have made some changes that have also been made on the other branch.

**Pull request closed without merging**